src/utils/kubernetes.ts

```diff
@@ -0,0 +1,153 @@
+import {KubeConfig, CoreV1Api, V1Namespace} from '@kubernetes/client-node';
+import {ActionOnInvalid} from '@kubernetes/client-node/dist/config_types';
+import {
+   extension as kubernetesExtension,
+   HelmV1,
+   KubectlV1
+} from 'vscode-kubernetes-tools-api';
+import {Errorable} from './errorable';
+import {withTempFile} from './temp';
+
+kubernetesExtension.kubectl.v1;
+
+export interface KubernetesApi {
+   listNamespaces(): Promise<Errorable<V1Namespace[]>>;
+   createNamespace(name: string): Promise<Errorable<void>>;
+   applyManifests(manifestPaths: string[]): Promise<Errorable<string>>;
+   applyKustomize(kustomizationDirectory: string): Promise<Errorable<string>>;
+   installHelm(chartDirectory: string): Promise<Errorable<string>>;
```

Comment:
💡 just fyi - please do check, If we have taken dependency on `Kubernetes extension` I think helm install comes with it, so installing helm at this extension might not be needed. Thanks.

Reply:
Aligned on that! We do take dependency on the Kubernetes extension and we do get the Helm install from them. You are completely correct.